package.json

```diff
@@ -8,7 +8,9 @@
   },
   "description": "Shared renovate configuration and GitHub Action",
   "scripts": {
-    "test": "renovate-config-validator default.json",
+    "updatePackageRules": "node ./updatePackageRules.js && npm run lint:fix", 
```

Comment:
Is there a better name for this script? Do we typically use camel-case in package.json?

Also should I move the .js to a `scripts/` subdir or similar for clarity?

Reply:
We normally use snake-case for filenames, for script names in package.json there's no particular pattern but I think this would normally come under `build`, ie `npm run build` since that's what it's essentially doing in this context